cmd/fortifyExecuteScan_test.go

```diff
@@ -464,7 +516,8 @@ func TestExecutions(t *testing.T) {
 			utils := newFortifyTestUtilsBundle()
 			influx := fortifyExecuteScanInflux{}
 			auditStatus := map[string]string{}
-			reports, _ := runFortifyScan(ctx, data.config, &ff, &utils, nil, &influx, auditStatus)
+			execInPath = mockExecinPath
+			reports, _ := runFortifyScan(ctx, data.config, &ff, utils, nil, &influx, auditStatus)
```

Comment:
Please see the diff here and fix the tests. It should be &utils. Change it in every call of runFortifyScan